src/iexamine.cpp

```diff
@@ -857,6 +857,15 @@ void iexamine::bars(player &p, const tripoint &examp)
     p.setpos( examp );
 }
 
+void iexamine::deployed_furniture( player &p, const tripoint &pos )
+{
+    p.add_msg_if_player( m_info, string_format( _( "You take down the %s." ),
```

Comment:
You don't need to do the string formatting here, the `add_msg` functions do string formatting on their own: `add_msg( "format", ... );`